doc/code/qml_pauli.rst

```diff
@@ -74,6 +73,32 @@ array([[ 0. +0.j,  0. +0.j,  0.5+0.j,  0. +0.j],
        [-0.5+0.j,  0. +0.j,  0. +0.j,  0. +0.j],
        [ 0. +0.j,  0.5+0.j,  0. +0.j,  0. +0.j]])
 
+We can intuitively use Pauli arithmetic to construct Hamiltonians consisting of :class:`~pennylane.pauli.PauliWord`
+and :class:`~pennylane.pauli.PauliSentence` objects like the spin-1/2 XXZ model Hamiltonian,
+
+.. math:: H_\text{XXZ} = \sum_j [J^\bot (X_j X_{j+1} + Y_j Y_{j+1}) + J^\text{ZZ} Z_j Z_{j+1} + h Z_j].
+
+Here we look at the simple topology of a one-dimensional chain with periodic boundary conditions (i.e. qubit number :math:`n+1 \equiv 1`).
+In code we can do this via the following example with 4 qubits.
+
+.. code-block:: python3
+
+    n_wires = 4
+    J_orthogonal = 1.5
+    ops = [J_orthogonal * (PauliWord({i:"X", (i+1)%n_wires:"X"}) + PauliWord({i:"Y", (i+1)%n_wires:"Y"})) for i in range(n_wires)]
```

Comment:
This line is much longer than the code block in the docs - could we break it at the `+`, or something similar?

Reply:
✅